vendor/brave-ios/Ledger/Generated/NativeLedgerClient.mm

```diff
@@ -98,7 +98,7 @@
 void NativeLedgerClient::OnRecoverWallet(ledger::Result result, double balance, std::vector<ledger::GrantPtr> grants) {
   [bridge_ onRecoverWallet:result balance:balance grants:std::move(grants)];
 }
-void NativeLedgerClient::OnRemoveRecurring(const std::string & publisher_key, ledger::RecurringRemoveCallback callback) {
+void NativeLedgerClient::RemoveRecurringTip(const std::string & publisher_key, ledger::RemoveRecurringTipCallback callback) {
   [bridge_ onRemoveRecurring:publisher_key callback:callback];
```

Comment:
(when rename happens in `NativeLedgerClientBridge.h`, this will have to be updated to match)

Reply:
Oh and when name changes here it will also have to change in BATBraveLedger.mm where my other review comment is